app/src/main/java/mx/unam/fciencias/moviles/practica_2/MainActivity.java

```diff
@@ -12,6 +12,11 @@
  * <a href="cris_rgj13@hotmail.com">cris_rgj13@hotmail.com</a>
  **/
 
+/**
+ * @author Ximena Andrade Luviano.
+ * <a href="mailto:andradeluvianoximena@ciencias.unam.mx">andradeluvianoximena@ciencias.uanm.mx</a>
+ * */
```

Comment:
Cuando resolvieran el conflicto las etiquetas de autoría debían quedar dentro del mismo comentario.
```suggestion
/**
 * @author Ximena Andrade Luviano.
 * <a href="mailto:andradeluvianoximena@ciencias.unam.mx">andradeluvianoximena@ciencias.uanm.mx</a>
 * @author Gonzalez Jimenez Cristian Rafael
 * <a href="cris_rgj13@hotmail.com">cris_rgj13@hotmail.com</a>
 * */
```